kiichain-pay/introduction/generating-api-keys.md

```
@@ -1,2 +1,257 @@
+After you close the dialog, the key appears in the list showing only a **masked prefix** (`abc123def456***`) and its public key. From the key's card you can **copy the public key**, **rotate**, or **delete** the key.
+
+<figure><img src="../../.gitbook/assets/kiichain-pay-api-key-card.png" alt="API key card with scopes expanded"><figcaption><p>An existing key: masked value, the expanded <strong>Scopes</strong> list, and the copy / rotate / delete actions.</p></figcaption></figure>
+
+- **Rotate** issues a new API key and private key for the same key entry (the old credentials stop working immediately). The new secret is shown once, exactly like creation.
+- **Delete** revokes the key immediately.
+
+## 3. Authenticate your requests
+
+Send your API key as an `Authorization` header with the **`APIKey`** scheme (not `Bearer`):
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)